### PR TITLE
Update metadata documentation for the latest changes

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
@bparees @adellape PTAL

This is reflecting the recent changes made to all our Docker images and also  https://github.com/openshift/origin/blob/master/docs/proposals/metadata.md

**Comment by bparees** [comment]
lgtm.

File: creating_images/metadata.adoc [hunk 1]
```diff
@@ -15,23 +15,25 @@ allowing OpenShift to create a better experience for developers using your
...
+variable in that they are not visible to the running application and they can
+also be used for fast look-up of images and containers.
+
+https://docs.docker.com/reference/builder/#label[Docker documentation] for more
+information on the `LABEL` instruction.
+
+The label names should typically be namespaced. The namespace should be set
+accordingly to reflect the project that is going to pick up the labels and use
+them. For OpenShift the namespace should be set to *_openshift.io/_* and for
+Kubernetes the namespace is _*k8s.io/*_. For simple labels, like `displayName`
```
Comment (adellape) [inline creating_images/metadata.adoc]: The underscores and asterisks are backwards here.

**Comment by adellape** [comment]
@mfojtik :cool:, just the 1 comment.

**Comment by mfojtik** [comment]
@adellape thanks! updated

**Pull request merged**